Tests/SentryTests/Integrations/MetricKit/SentryMetricKitIntegrationTests.swift

```diff
@@ -85,6 +85,25 @@ final class SentryMetricKitIntegrationTests: SentrySDKIntegrationTestsBase {
         }
     }
     
+    func testAttachDiagnosticAsAttachment() throws {
```

Comment:
m: Should we have a test to ensure attachment is not send if `enableMetricKitRawPayload` is false?